docs-ref-conceptual/get-started-tutorial-3-use-variables.md

```diff
@@ -0,0 +1,305 @@
+echo "My new Azure Kev Vault ID is $myNewKeyVaultID"
+
+# Create a new secret returning the secret ID
+kvSecretName=<myKVSecretName>
+kvSecretValue=<myKVSecretValue>
+myNewSecretID=$(az keyvault secret set --vault-name $kvName --name $kvSecretName --value $kvSecretValue --query id --output tsv)
+echo "My new secret ID is $myNewSecretID"
+
+# Reset your default output to json
+az config set core.output=json
+```
+
+# [PowerShell](#tab/powershell)
+
+```azurecli-interactive
+# Set your variables.
+$randomIdentifier=(New-Guid).ToString().Substring(0,8)
+$rgName="<msdocs-tutorial-rg-00000000>"
+$kvName="msdocs-kv-$randomIdentifier"
+$location="eastus"
+
+# Set your default output to none
+az config set core.output=none
+
+# Create a new Azure Key Vault returning the Key Vault ID
+$myNewKeyVaultID=$(az keyvault create --name $kvName --resource-group $rgName --location $location --query id --output tsv)
+echo "My new Azure Kev Vault ID is $myNewKeyVaultID"
+
+# Create a new secret returning the secret ID
+$kvSecretName="<myKVSecretName>"
+$kvSecretValue="<myKVSecretValue>"
+$myNewSecretID=$(az keyvault secret set --vault-name $kvName --name $kvSecretName --value $kvSecretValue --query id --output tsv)
+echo "My new secret ID is $myNewSecretID"
+
+# Reset your default output to json
+az config set core.output=json
+```
+
+---
+
+## Get the contents of a JSON file and store it in a variable
+
+This next section is a "stretch task" for an onboarding tutorial. However, to manage Azure resources in a development/stage/production scenario, you need to be able to read a configuration file.
+
+Are you ready to stretch your Azure CLI skills? Create a JSON file containing the following JSON, or your file contents of choice. Save the text file to you local drive. If you are working in Azure Cloud Shell, use the `upload/download files` icon in the menu bar to store the text file in your cloud storage drive.
+
+```text
+{
+  "environments": {
+    "dev": [
+      {
+        "id": "1",
+        "kv-secretName": "dev1SecretName",
+        "status": "inactive",
+      },
+      {
+        "id": "2",
+        "kv-secretName": "dev2SecretName",
+        "status": "active"
+      }
+    ],
+    "stg": {
+      "id": "3",
+      "kv-secretName": "dev3SecretName"
+    },
+    "prod": {
+      "id": "4",
+      "kv-secretName": "dev4SecretName"
+    }
+  }
+}
+```
+
+Store the contents of your json file in a variable for further use in your Azure CLI commands. In this example, change `msdocs-tutorial.json` to the name of your file. Do not save the `echo` command in production-level scripts as the output is saved in your log file.
+
+# [Bash](#tab/bash)
+
+This Bash script was tested in [Azure Cloud Shell](/azure/cloud-shell/overview) and depends on the Bash [jq](https://jqlang.github.io/jq/manual/) command.
+
+```azurecli-interactive
+# Show the contents of a file in the console
+fileName=msdocs-tutorial.json
+cat $fileName | jq
```

Comment:
I keep getting "jq command not found" when I run it in gitbash. I can get it to show the JSON contents successfully but anything that has "jq" in it returns that error.

Reply:
Your environment must have jq preinstalled as it "depends on the Bash [jq]() command".  However, if you missed this, let me put in a note with your error message.